vertx-template-engines/vertx-web-templ-mvel/src/main/java/io/vertx/ext/web/templ/impl/MVELTemplateEngineImpl.java

```diff
@@ -67,11 +70,16 @@ public void render(RoutingContext context, String templateFileName, Handler<Asyn
       }
       Map<String, RoutingContext> variables = new HashMap<>(1);
       variables.put("context", context);
-      handler.handle(Future.succeededFuture(Buffer.buffer((String)TemplateRuntime.execute(template, variables))));
+      String directoryName = Paths.get(templateFileName).getParent().toAbsolutePath().toString();
```

Comment:
Should we use `VertxInternal.resolveFile` instead so it will work both for file system and files bundled in a fat jar?